Gui/opensim/build_id.properties.in

```diff
@@ -0,0 +1 @@
+netbeans.buildnumber=@BUILD_ID@
```

Comment:
Is there a reason for using `buildnumber` in one place and `BUILD_ID` in another? It would be nice to use the same name in both places if possible.